src/SparceCoverageCollector.js

```diff
@@ -0,0 +1,119 @@
+class SparceCoverageCollector {
```

Comment:
I consider them as quite handy syntactic sugar and prefer using them over es5 approach or doing some own helpers…

And I like using them for cases where more object oriented approach looks suitable(for example this one)